scripts/transforms/update_workflow_tags.sh

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-if [[ $# != 2 ]]; then
+if [[ $# != 3 ]]; then
   cat << EOF
-	"Incorrect number of parameters provided. The required parameters are versions_file and pipeline_path. 
+	$1 $2 
```

Comment:
should we add more explanation here? instead just $1 and $2

Reply:
Thanks I removed the redundant printing...